src/cli.ts

```diff
@@ -14,11 +15,62 @@ if (configurations.email.enabled) {
 
 import { sync } from './database';
 import courseController from './features/courses/course-controller';
+import appSequelize from './database/app-sequelize';
+import StudentWorkbook from './database/models/student-workbook';
+import rendererHelper from './utilities/renderer-helper';
 
-(async (): Promise<void> => {
-    await sync();
-
+const syncMissingGrades = async (): Promise<void> => {
     logger.info('Performing missing grade sync');
     await courseController.syncMissingGrades();
     logger.info('done');
+};
+
+const cleanupWorkbooks = async (): Promise<void> => {
```

Comment:
This is what will retroactively go and cleanup the existing data, it has been tested on 26263 records